app/views/course/achievements/_form.html.slim

```diff
@@ -5,6 +5,10 @@
 
   = f.input :description
 
+  = f.label t('.badge')
+  div.achievement-form = display_achievement_badge(@achievement)
+  = f.file_field :badge, class: 'upload-btn'
```

Comment:
For now, only course_user image, and achievement badges have such an upload button.  

There is 1 more instance where this will be used, which is the course logo. Once we do that, then I standardise? All logos are linked in #624 

We will probably need to have a centralised place to handle styling of forms where we can do this.